src/zeit/retresco/configure.zcml

```diff
@@ -44,4 +45,11 @@
     title="Update TMS index"
     />
 
+  <browser:page
```

Comment:
Das gehört dann in die configure.zcml für das json-Untersverzeichnis, und die muss man dann in der [site.zcml](https://github.com/zeitonline/vivi-deployment/blob/master/components/zope/site.zcml#L49) anziehen.